overlay/base/usr/lib/system/_os/cli/checkupdates.py

```
@@ -54,18 +54,17 @@ def command(args: Namespace) -> None:
+    if cast(bool, args.download) and (
+        [x for x in updates if x.startswith(f"{baseImage()} ")]
+        and force
+        or pull_available()
+    ):
+        try:
+            pull()
 
-            except BaseException:
-                traceback.print_exc()
-                sys.exit(1)
+        except BaseException:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `BaseException`

(BLE001)